### ISSUE TITLE
There is no Keyword Match Type in GA4

### ISSUE THREAD

**Issue by varelaerick** [googleanalytics/ga-dev-tools#1053]
I'm tring to analyze the `GoogleAdsQuery` in GA4 with and without the Query Explorer, however, I'm frustrated in try figuring out precisely what Keyword brought the `googleAdsQuery`. The dimension `googleAdsKeyword` led me to the words or phrases that I choose to get my ad in front of the customers but do not bring the Match Type. 

So if I have the same _word_ or _phase_ in the Exact Match Type and Phase Match Type for example, the GA4 will bring the same Keyword for both of them, not specifying which exact was the keyword with the match type.

In GAU it was possible to do this kind of analysis because of the feature `ga:adKeywordMatchType ` but this dimension does not exist yet in the GA4 which makes it impossible to figure out what Keyword with Match Type was responsible for the GoogleAdsQuery.

To solve that situation the creation of a new dimension in GA4 `googleadsKeywordMatchType` with the values (Phrase, Exact, or Broad) would be perfect. 

Another alternative would be to bring the complete `googleAdsKeyword` with the characters that define the kind of Match Type. E.g `"basket ball"` for phase or `[basket ball]` for exact and` basket ball` for broad.

I thank you for your attention and will be happy with any solution.

**Comment by jamesmnixon** [comment]
Agreed! The lack of visibility into the keyword and search term match type for that matter is causing huge problems for how we display client data. Please add those reportable fields back in for GA4.
